inc/datahandlers/user.php

```diff
@@ -1536,17 +1536,24 @@ function delete_content($delete_uids=false)
 		$plugins->run_hooks('datahandler_user_delete_content', $this);
 
 		$db->delete_query('userfields', "ufid IN({$this->delete_uids})");
-		$db->delete_query('privatemessages', "uid IN({$this->delete_uids})");
+		$db->delete_query('privatemessages', "uid IN({$this->delete_uids}) OR fromid IN({$this->delete_uids})");
```

Comment:
Ok. Added it here then: https://github.com/mybb/mybb/issues/1820 and will revert the OR.